staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -1049,6 +1063,33 @@ func (cfgCtlr *configController) maybeReapReadLocked(plName string, plState *pri
 	cfgCtlr.configQueue.Add(0)
 }
 
+// PriorityLevelMaxSeats returns a functions invoked by the work estimator
+// to dynamically determine max seats for a request in a given priority level.
+// See priorityLevelMaxSeatsLocked for more details.
+func (cfgCtlr *configController) PriorityLevelMaxSeats() fcrequest.MaxSeatsFunc {
```

Comment:
Sorry for not being clear - that's not what I wanted. What I wanted is:
```
func (cftCtlr *configController) PriorityLevelMaxSeats(plName string) uint64) {
  return cftCtlr.priorytLevelMaxSeatsLocked(plName)
}
```
as it was

this function now implements the MaxSeatsFunc interface, so you would be good can get rid of `()` in few places

Reply:
All good, I reverted this change now